content/en/blog/2020/new-deployment-model/index.md

```diff
@@ -0,0 +1,64 @@
+---
+title: A New Istiod Deployment Model
+subtitle: Clear Separation between Mesh Operator and Mesh Users
+description: A new deployment model for Istio.
+publishdate: 2020-08-24
+attribution: "Lin Sun (IBM), Iris Ding (IBM)"
+keywords: [istiod,deployment model,central istiod,install,deploy,'1.7']
+---
+
+## Background
+
+From experience working with various service mesh users and vendors, we believe there are 3 key personas for a typical service mesh:
+
+* Mesh Operator, who manages the service mesh installation and upgrade.
+
+* Mesh Users:
+
+  1. Platform Owner, who owns the service mesh platform and defines the overall strategy and implementation for service owners to adopt service mesh.
```

Comment:
I'm not sure this breakdown is right. There are definitely "platform owners", and "service owners".  Is a "mesh operator" different to a platform owner?  I wouldn't call them a mesh user, I'd say they are the team that manages Kubernetes and makes it available to others.

I tend to break personas down as: platform owner, service operator, app developer.

Reply:
What is the purpose of describing 3 different roles in this blog? Other than the fact that there are mesh operators vs. others (2 things to separate), this deployment model doesn't support any further breakdown of roles, so describing the sub roles for configuring vs implementing services on the data-plane side has nothing to do with what this blog is providing. Or am I missing something?

Reply:
You are right, the benefit introduced by this blog doesn't differ much for the 2 sub roles (under mesh admin/user).  It is described here for completeness of common mesh personas.

Reply:
the key for service owner role is their experience with Istio mesh stay the same, whether they are using central istiod or not.